src/components/PracticeAdvice/PracticeAdvice.jsx

```diff
@@ -19,7 +19,7 @@ function PracticeAdvice() {
             the two major realms of Classical physics: Newtonian mechanics
           </p>
         </div>
-        <div className="practiceBlocks">
+        <div className="practice__blocks">
```

Comment:
bad - relates to practice-blocks, but contains sth inside, so it should be ```practice-blocks__container``` - or - ```practice-blocks__wrapper```. 

it could be ```practice__block```, but practice__block is what we've got inside this element. 
So, if inside this element we've got practice__block, then this element acts as a container or wrapper